sqladapter/src/main/java/com/ncc/neon/adapters/sql/SqlQueryConverter.java

```diff
@@ -268,16 +269,20 @@ public static String convertMutationIntoInsertQuery(MutateQuery mutateQuery) {
     }
 
     public static String convertMutationQueryIntoDeleteQuery(MutateQuery mutateQuery) {
+        String whereClause = getWhereClause(mutateQuery);
+        String sqlQueryString = "DELETE FROM " + mutateQuery.getDatabaseName() + "." +
+                mutateQuery.getTableName() + " WHERE " + whereClause;
+
+        return sqlQueryString;
+    }
+
+    private static String getWhereClause(MutateQuery mutateQuery) {
```

Comment:
Can we rename this method to be a little more specific?  Maybe something like `getWhereClaseFromMutateQuery`?

Reply:
Fixed.